src/common/__tests__/user-store.test.ts

```diff
@@ -45,18 +45,6 @@ describe("user store tests", () => {
       expect(us.seenContexts.has("bar")).toBe(true);
     });
 
-    it("allows setting and getting preferences", () => {
-      const us = UserStore.getInstance<UserStore>();
-
-      us.preferences.httpsProxy = "abcd://defg";
-
-      expect(us.preferences.httpsProxy).toBe("abcd://defg");
-      expect(us.preferences.colorTheme).toBe(UserStore.defaultTheme);
-
-      us.preferences.colorTheme = "light";
-      expect(us.preferences.colorTheme).toBe("light");
-    });
-
```

Comment:
Why was this test removed?

Reply:
I moved it to a separate `preferences` test suite,
https://github.com/lensapp/lens/pull/2146/files#diff-6d7101eddd71988a42a32649e56b79e1afdaf7d7f60d005024d4ca2a06d47f40R68